### PR TITLE
Support datetime

### PR DESCRIPTION
**Pull request by syucream**
Support datetime field.

image

image

### CODE REVIEW

File: apiclient/typescript-fetch/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmm-com/airone-apiclient-typescript-fetch",
-  "version": "0.0.17",
+  "version": "0.0.18",
```
Comment (syucream) [inline apiclient/typescript-fetch/package.json:3]: Need to support datetime type.
@userlocalhost @hinashi could you release it if no problem?

**Comment by userlocalhost** [comment]
It's great to support old UI compatibility as Date type attribute 👍 👍 
スクリーンショット 2024-06-10 15 46 40

**Comment by userlocalhost** [comment]
@syucream 
I found a small glitch about it that occurs an exception just after I press `BackSpace` key on it. Would you please fix it if it's not so hard?
スクリーンショット 2024-06-10 15 45 07
スクリーンショット 2024-06-10 15 45 20

**Comment by syucream** [comment]
@userlocalhost Fixed that, with enhancing validation logic for `date` type!

**Review by userlocalhost — APPROVED** [review]
@syucream Excellent!! Thank you very much for making this robust validation processing 👍

**Review by hinashi — APPROVED** [review]
LGTM!

**Pull request merged**